### DISCUSSION TITLE
Example in GitHub docs causes No such file or directory

### DISCUSSION THREAD

**Discussion by mmtg5** [community/community#43985] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi there,

I tried "Setting an output parameter" with "Environment files" as described in GitHub docs https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-3

Unfortunately this causes an error 

> /home/runner/work/_temp/_runner_file_commands/set_output_[....]: No such file or directory

**Comment by jsoref** [comment]
Are you using GitHub or GHES or GHEC? (Which version if GHES?)
Are you using a self-hosted runner?
What OS image are you using? 

Did you try a job whose only step is to create a `hello world` output?

**Reply by mmtg5** [reply]
I'm using GitHub and the ubuntu-latest.

Yes, the Hello World is fine and the environment variable https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable is also fine

The only issue is with the output parameter https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

**Reply by jsoref** [reply]
I mean as the only step:

```
echo "hello=world" >> $GITHUB_OUTPUT
```

My guess is that you're deleting `/home/runner/work/_temp/runner_file_commands` in one of your steps

**Reply by mmtg5** [reply]
No, I tested the only step like this:

jobs:
   first:
      runs-on: ubuntu-latest
      steps:
      - name: hello world
         run: echo "hello=world" >> $GITHUB_OUTPUT

no steps before, none after. 1 job "first" and one step "hello world"

the result is

> | /home/runner/work/[....]/workflow/0: line 1: /home/runner/work/_temp/_runner_file_commands/set_output_[....]: No such file or directory

**Reply by mmtg5** [reply]
if I test the only step like this:

jobs:
first:
  runs-on: ubuntu-latest
  steps:
   - name: hello world
      run: echo "hello=world" >> $GITHUB_ENV

no steps before, none after. 1 job "first" and one step "hello world"

the result is

> ✅  Success - hello world

**Reply by jsoref** [reply]
Please use triple backticks to mark up your yaml here:

```
    ```yaml
    on:
      push:
    ```
```

I can't tell the difference between your two most recent posts.

It sounds like a basic test passes.

In which case, I'd insert `find /home/runner/work` steps at various points to find a step that's deleting directories it shouldn't.
